microcloud/cmd/microcloud/ask.go

```diff
@@ -593,7 +593,7 @@ func (c *CmdControl) askNetwork(sh *service.Handler, systems map[string]InitSyst
 
 	// Uplink selection table.
 	header := []string{"LOCATION", "IFACE", "TYPE"}
-	fmt.Println("Select exactly one network interface from each cluster member:")
+	fmt.Println("Select exactly one interface per system to dedicate for the distributed network:")
```

Comment:
Maybe if we say "distributed overlay networks" it's even more clear that the actual virtual network traffic is meant? Just an idea :)

Reply:
We need to mention that this interface is for traffic going between the overlay network and the ddyetbal uplink network. Otherwise it'll be understood its for intra cluster overlay traffic.

Reply:
How about:

```
Select an available interface per system to provide external connectivity for distributed network(s)
```

Reply:
The egress traffic is also a valid point. +1 for the last suggestion.

Reply:
Yeah, I used "external" as it could also be ingress too.

Reply:
And I used "connectivity" to imply Internet access, but not specifically mandate it.

Reply:
Should it be "network(s)"? The next questions mention "the distributed network".

Reply:
The user is selecting an interface for use by the distributed network(s) for external connectivity.
The init process is setting up one initial distributed network, but the user can add more in the future (thats one of the big benefits of OVN).